src/dotnet/commands/dotnet-migrate/MigrateCommand.cs

```diff
@@ -76,12 +86,57 @@ public int Execute()
 
             WriteReport(migrationReport);
 
+            MoveProjectJsonArtifactsToBackup(_doBackup, migrationReport);
```

Comment:
I realize this is not too easy, and doesn't need to happen for this PR.

But I do think that if the backup goes through ( or doesn't) it should be part of the report.